node/src/Commands.ts

```diff
@@ -1549,3 +1549,43 @@ export function createObjectIdletime(key: string): command_request.Command {
 export function createObjectRefcount(key: string): command_request.Command {
     return createCommand(RequestType.ObjectRefCount, [key]);
 }
+
+export type LPosOptions = {
+    rank?: number;
+    count?: number;
+    maxLength?: number;
+};
+
+function addLPosOptions(options: LPosOptions, args: string[]) {
```

Comment:
Some commands have a separate function to add options like we do here, and some do not (see createXAdd in this file). We might want to also align on whether we want a separate function or whether we want to add the options directly in the "createLPos" function like createXAdd does

Reply:
I've refactored so that the Options class will handle the logic.